### PR TITLE
[DOCS] TouchScreenButton

### PR DESCRIPTION
**Pull request by diowonderland**

### CODE REVIEW

File: doc/classes/TouchScreenButton.xml [hunk 1]
```diff
@@ -107,6 +120,7 @@
 			<argument index="0" name="bool" type="bool">
 			</argument>
 			<description>
+				If [code]true[/code], the button's shape is centered.
```
Comment (Noshyaar) [inline doc/classes/TouchScreenButton.xml]: `If [code]...[/code]` shouldn't be followed by comma.

File: doc/classes/TouchScreenButton.xml [hunk 2]
```diff
@@ -139,43 +156,57 @@
...
+			The button's texture for the pressed state.
 		</member>
 		<member name="shape" type="Shape2D" setter="set_shape" getter="get_shape">
+			The button's shape.
 		</member>
 		<member name="shape_centered" type="bool" setter="set_shape_centered" getter="is_shape_centered">
+			If [code]true[/code] the button's shape is centered.
 		</member>
 		<member name="shape_visible" type="bool" setter="set_shape_visible" getter="is_shape_visible">
+			If [code]true[/code] the button's shape is visibile.
```
Comment (Noshyaar) [inline doc/classes/TouchScreenButton.xml]: visible

File: doc/classes/TouchScreenButton.xml [hunk 3]
```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="TouchScreenButton" inherits="Node2D" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Button for touch screen devices.
 	</brief_description>
 	<description>
+		Button that can be visible on all or only on touch screen devices.
```
Comment (cbscribe) [inline doc/classes/TouchScreenButton.xml]: This sentence needs attention.  Needs work, but something like:

Button for touch screen devices. You can set it to be visible on all screens, or only on touch devices.

**Review by cbscribe — CHANGES_REQUESTED** [review]

**Comment by diowonderland** [comment]
Updated!

File: doc/classes/TouchScreenButton.xml [hunk 4]
```diff
@@ -139,43 +156,57 @@
 			<argument index="0" name="mode" type="int" enum="TouchScreenButton.VisibilityMode">
 			</argument>
 			<description>
+				Sets the button's visibility mode. See the [code]VISIBILITY_*[/code] constants.
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="action" type="String" setter="set_action" getter="get_action">
+			The button's action.
```
[the remark this answers is not in the document]
Comment (diowonderland) [reply doc/classes/TouchScreenButton.xml]: Open for suggestions :)

[the remark this answers is not in the document]
Comment (cbscribe) [reply doc/classes/TouchScreenButton.xml]: How about:

The button's action. Actions can be handled with [InputEventAction].

**Comment by diowonderland** [comment]
@cbscribe Done, changed the description for the action to your suggestion :)

**Review by cbscribe — APPROVED** [review]

**Comment by cbscribe** [comment]
Thanks for your contribution!

**Pull request merged**